js/script.js

```diff
@@ -1,29 +1,65 @@
-function calcSum() { 
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c1 = a + b;
-    document.getElementById('result').innerHTML = 'result ' + c1;   
- } 
- 
- function calcSub() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c2 = a - b;
-    document.getElementById('result').innerHTML = 'result ' + c2;   
- }
+let globalNumbers = document.querySelectorAll('.action-bar__numbers'),
+    operators = document.querySelectorAll('.action-bar__operators'),
+    cleanEntry = document.getElementById('ce'),
+    output = document.getElementById('output'),
+    xNumber = null,
+    yNumber = '',
+    operator = '',
+    operatorClicked = false;
+    output.value = '0';
 
- function calcMul() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c3 = a * b;
-    document.getElementById('result').innerHTML = 'result ' + c3;   
- }
+for (let i = 0; i < globalNumbers.length; i++) {
```

Comment:
You can extract the initialization of handlers to a separate function:

```
function initialize() {
  const globalNumbers = document.querySelectorAll('.action-bar__numbers');
  for (let i = 0; i < globalNumbers.length; i++) { ... }

  const operators = document.querySelectorAll('.action-bar__operators');
  for (let i = 0; i < operators.length; i++) { ... }

  ...
}
```

Then you will have to call this function somewhere in the script (e.g. on `document.onload` event).

It will help to encapsulate initialization logic (with all necessary variables) in a one simple function.